### ISSUE TITLE
Sponsors Page - Resize images

### ISSUE THREAD

**Issue by mattpen** [phetsims/tasks#470]
From [website/#298](https://github.com/phetsims/website/issues/298) - the new sponsor images need to be resized to improve delivery of the About Sponsors page.  The image sources can be found [here](https://docs.google.com/spreadsheets/d/1BPfMCA0-p5AB3sTYIDNqmKuikzjpE5MYsDSfQBXMR7A/edit#gid=0).  

The images should be sized to a max height of 150px and max width of 350px. The aspect ratio should not be changed.

Please upload the resized images [here](https://drive.google.com/drive/folders/0BwA8lLKUclCPd0R3bUZKSkdZdlU).  The filenames should be the same as the old filename but add `_350` to the end of the filename.  For example `wireless_generation_logo.png` would become `wireless_generation_logo_350.png`

**Comment by ariel-phet** [comment]
Hey @phet-steele - this one is for you :)

**Comment by phet-steele** [comment]
@mattpen the first link in the google doc for Hewlett Packard is the wrong image, can you find the right one and put it in there?

**Comment by mattpen** [comment]
I updated it with a link to the full image on the site.

**Comment by phet-steele** [comment]
@BryceAG @brroberts1231 since there isn't much in the way of testing, can you guys take a crack at this today? @BryceAG you were going to find a better way to do this to keep transparency, any ideas?

You can assign this back to me at the end of the day.

**Comment by brroberts1231** [comment]
@mattpen The logo for the King Saud University ECSME below (taken from the phet website)
![image](https://cloud.githubusercontent.com/assets/11947677/12686570/a1809bc6-c687-11e5-9463-b15e9a64a47c.png)
doesn't have a link to a full size image on the spreadsheet, it just links to a webpage. I could just upload the image I put here, but it's blurry. Not sure what to do with this one.

**Comment by mattpen** [comment]
If the links on the spreadsheet don't work, you can just go to the [sponsor page](https://phet.colorado.edu/en/about/sponsors), right click the image, and choose "Save image as".  The images on this page are the full size images, that's why we need to resize them.  

In the case of the KSU ECSME logo, the image is already appropriately sized.

**Comment by phet-steele** [comment]
@mattpen everything is resized! Switching over to you.

**Comment by ariel-phet** [comment]
@mattpen, quick ping on this issue. Might be nice to finish off and close

**Comment by mattpen** [comment]
The resized images were deployed to figaro tonight.  The page is still a little slow to load, but I think its due to the sheer volume of images on the page, rather than their size.  I only noticed the slowness if I spammed the Page Down button the instant the page appeared.

@ariel-phet is this acceptable?  Does the page need to be further optimized or is a QA task is needed?  Personally, I think this issue can be closed.
